odoo/tools/mail.py

```diff
@@ -40,7 +40,8 @@
      'data-o-mail-quote',  # quote detection
      'data-oe-model', 'data-oe-id', 'data-oe-field', 'data-oe-type', 'data-oe-expression', 'data-oe-translation-id', 'data-oe-nodeid',
      'data-publish', 'data-id', 'data-res_id', 'data-interval', 'data-member_id', 'data-scroll-background-ratio', 'data-view-id',
-     'data-class', 'data-mimetype',
+     'data-class', 'data-mimetype', 'data-shape', 'data-original-src', 'data-file-name', 'data-shape-colors',
+     'data-original-mimetype', 'data-original-id', 'data-resize-width',
```

Comment:
Aren't some of those attributes already used in stable version? Should not this fix target those stable vesions too?